evalml/pipelines/components/transformers/preprocessing/featuretools.py

```diff
@@ -141,14 +141,24 @@ def _handle_partial_dependence_fast_mode(self, X, pipeline_parameters):
             X (pd.DataFrame): Holdout data being used for partial dependence calculations.
             pipeline_parameters (dict): Pipeline parameters that will be used to create the pipelines
                 used in partial dependence fast mode.
+            target (str): The target whose values we are trying to predict. May be present in the
+                list of features in the DFS Transformer's parameters, in which case we should ignore it.
```

Comment:
It may be helpful to rephrase the second sentence here to make why we need it more explicit - i.e. "This is used to know which column to ignore if the target column is present in the list of features in the DFS Transformer's parameters" or something slightly cleaner sounding.

Reply:
I like this suggestion -- changing to it!